src/components/PWAInstructions/index.tsx

```diff
@@ -118,12 +119,21 @@ const devices = {
   },
 };
 
-const PWAInstructions = (props) => {
-  const [device, setDevice] = useState(null);
-  const [platform, setPlatform] = useState(null);
+type props = {
+  open: boolean;
+  handleDisplay: () => void;
+  deferredPWAPrompt: typeof BeforeInstallPromptEvent;
+};
+
+const PWAInstructions = ({ open, handleDisplay, deferredPWAPrompt }: props) => {
```

Comment:
Looks like you updated this to destructure `props`, but didn't update the code which is still trying to use the `props` object that doesn't exist anymore.